### DISCUSSION TITLE
stats number update

### DISCUSSION THREAD

**Discussion by ShorenaK** [anuraghazra/github-readme-stats#2582] [General]
My stats numbers are not updating; how can I fix it? It shows the same number daily regardless of how many commits I make.

**Comment by Zo-Bro-23** [comment]
We cache responses for around four hours by default. This means that stats will only update after four hours. If you want to avoid this for testing purposes you can add a random string to the URL: `?HDJS=hsjdj`. If you want more accurate stats that update more frequently, you can use the GRS GitHub Action (#2537) which we will be releasing soon.

**Comment by pchchv** [comment]
> We cache responses for around four hours by default. 

The stats haven't been updated for about a week now

**Comment by ShorenaK** [comment]
My stats are not updating at all, so I have 1,397 commits, and stats say 1.1k for two weeks. Is there a way I can display the whole number of 1,397 commits?

**Reply by Zo-Bro-23** [reply]
Commits would be for the last 365 days unless the `show_all_commits` option is enabled. Right now you can't display the entire number, but an option can be added once #2409 gets merged (I'm working on it).
